lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -317,11 +317,11 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 12, 7),
-        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 12, 7),
+        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 13, 8),
```

Comment:
It's a shame that these numbers have gone up, even if only by one call. Is this just inevitable given the new lookup from the database cache?